util/settings/settings_test.go

```diff
@@ -31,5 +31,91 @@ func TestUpdateSettingsFromConfigMapResourceExclusions(t *testing.T) {
 	err = updateSettingsFromConfigMap(&settings, &configMap)
 
 	assert.NoError(t, err)
-	assert.Equal(t, []ExcludedResource{{APIGroups: []string{}, Kinds: []string{}, Clusters: []string{}}}, settings.ResourceExclusions)
+	assert.Equal(t, []FilteredResource{{APIGroups: []string{}, Kinds: []string{}, Clusters: []string{}}}, settings.ResourceExclusions)
+}
+
+func TestUpdateSettingsFromConfigMapResourceInclusionsAddInclusion(t *testing.T) {
```

Comment:
Ah! We'll have a merge conflict here as I've made it easier to test this [see code](https://github.com/argoproj/argo-cd/pull/1496/files#diff-c6b0680a5f2fe0ed5aa9197202e013e5R24.). But great work anyway. Now, lets race to see who's PR gets merged first!

Reply:
🏃‍♂️🏃‍♂️🏃‍♂️🏃‍♂️